### PR TITLE
[views] "expo-views" - a module for common views like View, ScrollView, Text, etc...

### PR DESCRIPTION
**Pull request by ide**
This provides a common interface across more than just Android and iOS and also sets up the API to more easily add in a more optimized image implementation, for example. This is intentionally a very thin module that should not change the behavior of any apps and helps restructure the code to be more universal.

### CODE REVIEW

File: packages/expo-views/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Common views that are used for Expo apps that run on Android, iOS, and web. This is similar to [`react-primitives`](https://github.com/lelandrichardson/react-primitives) or [`@emotion/primitives`](https://emotion.sh/docs/@emotion/primitives) for Expo.
+
+This exports five components:
+
+- View
+- ScrollView
+- Text
+- TextInput
+- Image
```
Comment (tsapeta) [inline packages/expo-views/README.md]: What about StyleSheet?

**Review by tsapeta — APPROVED** [review]

File: packages/expo-views/README.md (hunk 1 above)
Comment (ide) [reply packages/expo-views/README.md]: 😱

**Review by brentvatne — CHANGES_REQUESTED** [review]
the world is not ready yet

**Comment by ide** [comment]
I have a better approach in mind that is more modular and simpler to name.

**Pull request closed without merging**